source/init/lodash-memoize.ts

```diff
@@ -0,0 +1,3 @@
+import memoize from 'lodash/memoize'
+
+memoize.Cache = WeakMap
```

Comment:
When filtering the student orgs list via the searchbar we received a red screen with the following message:

```console
TypeError: Attempted to set a non-object key in a WeakMap
```

<img src="https://user-images.githubusercontent.com/5240843/166167295-8d4e5790-1b06-4611-8a6b-1554e1cd2b4f.png" width=400 />

Reply:
gdi I forget that weakmaps can't do primitives for keys. We'll have to revert this change.

Reply:
What about setting it the way course search sets it?

https://github.com/StoDevX/AAO-React-Native/blob/8b5845dc921b79d802eece2fcb032f85627feee6/source/views/sis/course-search/list.tsx#L57-L60